Gemfile

```diff
@@ -0,0 +1,16 @@
+source "https://rubygems.org"
+
+gem 'autoproj', git: 'https://github.com/rock-core/autoproj'
+group :vscode do
+    gem 'pry'
```

Comment:
You need to add rubocop-rock here, or it won't be installed and rubocop will (silently) fail in vscode. There's quite a bit of rubocop violations because of that.

Reply:
Good catch.. Thanks

Reply:
I was surprised with some of the style errors, which I new rubocop was warning about.

Reply:
Yeah, there are probably a lot of them. Just a heads up: the same is missing in autoproj-ci as well..

Reply:
> Yeah, there are probably a lot of them. Just a heads up: the same is missing in autoproj-ci as well..

Oh .. Thanks